mlflow/utils/async_logging/async_logging_queue.py

```diff
@@ -23,6 +24,22 @@
 _logger = logging.getLogger(__name__)
 
 
+ASYNC_LOGGING_WORKER_THREAD_PREFIX = "MLflowBatchLoggingWorkerPool"
+ASYNC_LOGGING_STATUS_CHECK_THREAD_PREFIX = "MLflowAsyncLoggingStatusCheck"
+
+
+class QueueStatus(enum.Enum):
+    """Status of the async queue"""
+
+    # The queue is running worker threads to consume the new data.
```

Comment:
worker thread is a little confusing here - maybe we should say something like:
```
# The queue is listening to new data and logging enqueued data to MLflow.

# The queue is not listening to new data, but still logging enqueued data to MLflow.

# The queue is neither listening to new data or logging enqueued data to MLflow.
```

Reply:
Sounds good, that's def more clear!